def/es6-modules.js

```diff
@@ -0,0 +1,80 @@
+var types = require("../lib/types");
+var def = types.Type.def;
+var or = types.Type.or;
+var defaults = require("../lib/shared").defaults;
+var builtin = types.builtInTypes;
+var isString = builtin.string;
+
+// Based on https://github.com/estree/estree
+
+
+def("ModuleSpecifier2")
+    .bases("Node")
+    .build("local")
```

Comment:
Does this need to be buildable? If not, you can just remove the `.build("local")` line to make the type "abstract" (i.e. other types can inherit from it, but there won't be a builder called `builders.moduleSpecifier2`).